ui/src/main/webapp/src/app/shared/upload/alternative-upload-queue.component.scss

```diff
@@ -26,6 +26,7 @@ li {
 .progress-bar {
   padding-top: 5px;
   padding-bottom: 5px;
+  color: #363636;
```

Comment:
For me, this is quite a bit harder to read than the previous colors. Also, having the completion indicator move around is visually jarring to me.